SeanLee.md

```diff
@@ -1,38 +1,31 @@
 ### Last week
 
+* [Study]
+  - Dialer API - Study TelephonyHelper.js and Telephony related APIs.
+
 * [SIM Lock]
-  - [code reviewing][Bug 1111411] Resolve SIM lock issue
-  - [WIP][Bug 1119118] l10n issue. Verified by partner. review code as soon.
+  - [CLOSED][Bug 1111411] Resolve SIM lock issue
+  - [code reviewing][Bug 1119118] l10n issue. Verified by partner. review code as soon.
 
 * [STK]
-  - [code reviewing][Bug 1119640] STK setup event issue. Location Status PASS.
-  - [CLOSED][Bug 1118677] Partner’s code issue. RESOLVED INVALID
-  - [CLOSED][Bug 1118701] Provide a trail patch to show a notification when the reset type of STK_CMD_REFRESH is received. partner have not defined the behavior, so it is closed.
-
-* [Other]
-  - STK TLV parser tool -  This tool can help people to read radio.log more easily.
-
-* [Partner Support]
-  - On-site support to fix SIM lock and STK Setup Event issue. 2014/1/6 ~ 2014/1/10
-
-* [Dialer]
-  - [WIP][bug 1118643][Bug 1118651] Emergency call dial tone issue (see also Bug 834530)
-
-* [Contacts]
-  - [CLOSED][bug 1115628] Resolved by other bug.
+  - [CLOSED][Bug 1119640] STK setup event issue. Location Status PASS.
+  - [CLOSED][Bug 1119633] String issues. Partner test PASS.
+  - [CLOSED][Bug 1119135] Duplicate to 1119633 .
+  - [CLOSED][Bug 1121301] redundant event envelope of setup event. Gaia remove some specific events.
 
 ### This week
 
-* [SIM Lock]
-  - [code reviewing][bug 1111411] Add unit test for SIM lock.
-  - [CLOSED][Bug 1120070] Partner code merging issue. Resolved Invalid.
+* [Study]
+  - Dialer API
```

Comment:
I am studying the following topics which are related to Dialer App/API
1. MobileConnection
2. MMI
3. TelephonyHelper.js
4. Telephony/TelephonyCall APIs.